<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothWebViewConfig</class>
 <widget class="QWidget" name="SmoothWebViewConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>URL:</string>
     </property>
     <property name="buddy">
      <cstring>url</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="url"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Associated URL:</string>
     </property>
     <property name="buddy">
      <cstring>associatedUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="associatedUrl">
     <property name="toolTip">
      <string>URL to show in your browser when user clicks on &quot;maximize&quot;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Refresh periodically</string>
     </property>
     <property name="buddy">
      <cstring>refreshCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="refreshCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Update interval:</string>
     </property>
     <property name="buddy">
      <cstring>intervalSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KIntSpinBox" name="intervalSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Seconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Invert colors</string>
     </property>
     <property name="buddy">
      <cstring>invertColorsCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="invertColorsCheckBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Use Plasma background</string>
     </property>
     <property name="buddy">
      <cstring>plasmaBackgroundCheckbox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="plasmaBackgroundCheckbox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Hide scroll bars</string>
     </property>
     <property name="buddy">
      <cstring>hideScrollBarsCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="hideScrollBarsCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>210</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>refreshCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>intervalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>invertColorsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>plasmaBackgroundCheckbox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
